packages/aws-cdk/README.md

```diff
@@ -561,6 +561,8 @@ $ cdk bootstrap --show-template > bootstrap-template.yaml
 $ cdk bootstrap --template bootstrap-template.yaml
 ```
 
+If you have the need to encrypt the bootstrap artifacts, you can do this via the parameters `--bootstrap-ecr-key`.
```

Comment:
Can you update the readme to reflect the new state of the PR? It would also be helpful if you provided more information on the different options and the behavior.